driver/src/main/java/org/neo4j/driver/Config.java

```diff
@@ -780,7 +783,7 @@ public Config build()
     /**
      * Control how the driver determines if it can trust the encryption certificates provided by the Neo4j instance it is connected to.
      */
-    public static class TrustStrategy
+    public static class TrustStrategy implements Serializable
```

Comment:
Primarily for consistency and in case it gets extracted into a standalone class, would it make sense to add `serialVersionUID`?